tox.ini

```diff
@@ -5,6 +5,8 @@ envlist =
     py39-legacy-test
     py{39,310,311}-{poppydev,pysiafdev}-cov
 
+isolated_build = true
```

Comment:
```suggestion
```
this is no longer read by Tox 4: https://tox.wiki/en/latest/upgrading.html#isolated-environment-on-by-default